pkg/docker/index.html

```diff
@@ -28,6 +28,8 @@
     <link href="docker.css" type="text/css" rel="stylesheet" id="term-style">
     <script src="../base1/bundle.js"></script>
     <script src="../shell/shell.js"></script>
+    <script src="../storaged-client/bundle.js"></script>
```

Comment:
What is the storaged-client here. I don't it defined or refered  to anywhere else.